FirefoxPrivateNetworkVPN/Models/VPNLocations.swift

```diff
@@ -52,14 +52,31 @@ struct VPNCity: Codable {
         self.longitude = longitude
         self.servers = servers
         self.flagCode = flagCode
+        self.setServerSelection()
+    }
+
+    mutating func setServerSelection() {
+        var weightSum = 0
+        for server in servers {
```

Comment:
You can be more swifty with the reduce transform: https://developer.apple.com/documentation/swift/array/2298686-reduce